ee-feature-pack/galleon-feature-pack/pom.xml

```diff
@@ -237,12 +237,13 @@
             <scope>provided</scope>
         </dependency>
 
+        <!-- WFLY-19671. DO NOT include 'galleon content only' dependencies
```

Comment:
@bstansberry , I can see some remaining usages of wildfly-core-feature-pack-galleon-common (ban-transitive-deps and update-licenses-xml). BTW, perhaps cleaning wildfly-core-feature-pack-galleon-pruned too?

Reply:
@bstansberry , another point, just to be sure... Looking at https://github.com/wildfly/wildfly-core/tree/main/core-feature-pack/galleon-common/src/main/resources/packages/org.wildfly.openssl/pm/wildfly and https://github.com/wildfly/wildfly-core/blob/main/core-feature-pack/galleon-common/src/main/resources/packages/core.misc.common/pm/wildfly/tasks.xml we can see artifacts that require a version.

These artifacts are resolved thanks to the dependency in https://github.com/wildfly/wildfly-core/blob/main/core-feature-pack/common/pom.xml so we are good. 

Perhaps in wildfly-core we should add a note to not add dependencies to galleon-common, add them to common?

Reply:
@jfdenise re your second comment, I filed the much broader scoped https://issues.redhat.com/browse/WFCORE-6755. Thanks for pointing this out. I've long had the urge to get core-feature-pack reorganized but never had a reason to push it.

Reply:
Re your first comment -- good catch. I just pushed an update cleaning those out.